contrib_bots/bots/helloworld/helloworld.py

```diff
@@ -0,0 +1,38 @@
+# See readme.md for instructions on running this code.
+
+class HelloWorldHandler(object):
+    def usage(self):
+        return '''
+        This is a boilerplate bot that writes to the stream "Hello World!"
+
+        (Or just "beep boop")
+
+        This bot (along with Help) can be used as a template for other,
+        more sophisticated bots.
+        '''
+
+    def triage_message(self, message, client):
```

Comment:
The triage message function is obsolete and not required anymore. Feel free to just remove this chunk of code.